drivers/wifi/nrf700x/zephyr/src/qspi/inc/qspi_if.h

```diff
@@ -105,7 +105,6 @@ int func_rpu_sleep_status(void);
 #endif /* CONFIG_NRF_WIFI_LOW_POWER */
 
 #define QSPI_KEY_LEN_BYTES 16
-int qspi_configure_encryption(uint8_t *key);
-int qspi_enable_encryption(void);
+int qspi_enable_encryption(uint8_t *key);
```

Comment:
This might need a infocenter document update.

Reply:
Yes, but that is offline, so, unrelated to PR.